tests/core/component.test.js

```diff
@@ -129,6 +129,22 @@ suite('Component', function () {
       data = buildData(el, 'dummy', 'dummy', TestComponent.prototype.schema, 'red');
       assert.equal(data, 'red');
     });
+
+    test('multiple vec3 properties do not share same default value object', function () {
+      var data;
+      var el = entityFactory();
+      var TestComponent = registerComponent('dummy', {
+        direction: { type: 'vec3' },
+        position: { type: 'vec3' }
+      });
+      el.addEventListener('loaded', function () {
```

Comment:
this test is not actually doing anything because it is asynchronous and not using the `done` handler